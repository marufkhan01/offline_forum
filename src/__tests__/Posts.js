import React from 'react';
import { shallow, render } from 'enzyme';
import toJson from 'enzyme-to-json';

import Posts from '../components/Posts';



it('Posts component should render as expected',() =>{
    const component = shallow(<Posts />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});