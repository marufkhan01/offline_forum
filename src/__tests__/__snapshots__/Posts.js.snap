// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts component should render as expected 1`] = `
<div
  className="flex flex-wrap mx-auto pt-8"
>
  <CreateNewPost
    updatePosts={[Function]}
  />
</div>
`;
